netwerk/sctp/datachannel/DataChannel.cpp

```diff
@@ -369,6 +369,7 @@ DataChannelConnection::~DataChannelConnection() {
 }
 
 void DataChannelConnection::Destroy() {
+  // https://github.com/RecordReplay/backend/issues/3356
```

Comment:
I like including these comments

Reply:
Yeah, I'd like to avoid having bare calls to RecordReplayAssert or recordreplay::Diagnostic anywhere in gecko and the other runtimes.  Links like this makes them a lot easier to GC when closing an issue we needed diagnostics for, or doing more general cleanup like in this PR.

Reply:
Agreed. What is GC?

Reply:
Garbage collection (what I felt like I was manually doing while working on this patch).